frontend/tests/acceptance/pages/timeLogPage.js

```diff
@@ -9,13 +10,171 @@ module.exports = {
   elements: {
     timelog_title: '//h6[text()="Time Logs"]',
     empty_message: '//h5[text()="No timelogs added yet"]',
-    openaddtimelog_dialog: '//div[#aria-labelledby="Add time log"]',
+    openaddtimelog_dialog: '//button/span[contains(text(),"Add")]',
     openedittimelog_button: '//span[@aria-label="Edit time log"]',
-    addtimelog_button: '//button/span[contains(text(),"Add")]',
-    updatetimelog_button: '//button/span[contains(text(),"Update")]',
+    addtimelog_button: '//div[contains(@class,"MuiDialogContent-root")]//button/span[contains(text(),"Add")]',
+    updatetimelog_button: '//div[contains(@class,"MuiDialogContent-root")]//button/span[contains(text(),"Update")]',
     deletetimelog_button: '//span[@aria-label="Delete time log"]',
-    addtocalender_button: '//span[@label="Add to calender"]',
+    addtocalendar_button: '//span[@aria-label="Add to calender"]',
```

Comment:
```suggestion
    addtocalendar_button: '//span[@aria-label="Add to calendar"]',
```

Reply:
this should not be changed as it is the actual text in the UI.